migrations/20181004135102_add_custom_values_to_routing_condition_values.js

```diff
@@ -0,0 +1,18 @@
+const formatAlterTableEnumSql = require("../utils/migrateEnumChecks");
+exports.up = async function(knex) {
+  await knex.raw(
+    formatAlterTableEnumSql("Routing_Conditions", "comparator", [
+      "Equal",
+      "NotEqual",
+      "GreaterThan",
+      "LessThan",
+      "GreaterOrEqual",
+      "LessOrEqual"
+    ])
+  );
+  return knex.schema.table("Routing_ConditionValues", table => {
+    table.integer("customNumber").defaultsTo(null);
+  });
+};
+
+exports.down = function() {};
```

Comment:
Do we even need to implement this method? Can it be removed entirely?

Reply:
Annoyingly not, if we try remove this knex throws an error.

Reply:
Lodash has a `noop` function that may help convey semantically that `exports.down = noop`